src/commonTest/kotlin/baaahs/TestMovingHeadAdapter.kt

```diff
@@ -25,6 +25,9 @@ class TestMovingHeadAdapter(
     override val tiltRange: ClosedRange<Float> = toRadians(-135f)..toRadians(135f),
     override val tiltMotorSpeed: Float = 1f,
 
+    override val prismChannel: Dmx.Channel = TestChannel(5),
+    override val prismRotationChannel: Dmx.Channel = TestChannel(6),
```

Comment:
this conflicts with the shutterChannel number below, maybe bump that one up to 7 as well